add pages for building enterprise consent journey (#879)
codatio/codat-docs: docs/enterprise/tech-implementation/consent/uiexamples.md

---
title: "UI screens"

> description: "Sample screens for a product agnostic consent journey"

Reviewer: ```suggestion
description: "Sample screens for a product-agnostic consent journey"
```

---

This is an example flow for a consent journey which powers a business insights dashboard.

> * A user is prompted to select their accounting package

Reviewer: ```suggestion
* A user is prompted to select their accounting package.
```

> * They view the data they are sharing both in your platform and the accounting package

Reviewer: ```suggestion
* They view the data they are sharing both in your platform and the accounting package.
```

> * Throughout the journey they are reminded of why they are sharing data and the product value

Reviewer: ```suggestion
* Throughout the journey, they are reminded of why they are sharing data and the product's value.
```

> * There are links to get help should an issue arrise

Reviewer: ```suggestion
* The journey includes links enabling the user to get help should an issue arise.
```

> * Once linked they are able to manage their connection and choose to disconnect if they wish

Reviewer: ```suggestion
* Once linked, they are able to manage their connection and disconnect if they wish.
```

> * They are able to reauthorize from the same place if they change their mind

Reviewer: ```suggestion
* They are able to reauthorize from the same place if they change their mind.
```

> The screens here are for a business insights dashboard but these points are applicable for any use case.

Reviewer: ```suggestion
The example screens are for a business insights dashboard but these points are applicable to any use case. You can see this prototype in full screen [here](https://www.figma.com/proto/YWkKvsYgeHJskPsfuIpy7w/Codat-x-Generic-bank---Consent-Journey?page-id=601%3A4488&type=design&node-id=641-11421&viewport=1275%2C-2886%2C0.1&t=rrDznIIhmQ8EayyY-1&scaling=scale-down&starting-point-node-id=641%3A11421&show-proto-sidebar=1&mode=design).
```

<iframe  width="1500" height="950" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FYWkKvsYgeHJskPsfuIpy7w%2FCodat-x-Generic-bank---Consent-Journey%3Fpage-id%3D601%253A4488%26type%3Ddesign%26node-id%3D651-12998%26viewport%3D1275%252C-2886%252C0.1%26t%3DjQXc8Fg7a3kwfbK2-1%26scaling%3Dscale-down%26starting-point-node-id%3D641%253A11421%26show-proto-sidebar%3D1%26mode%3Ddesign" allowfullscreen>Loading demo..</iframe>

> If you wish to see this in full screen [click here](https://www.figma.com/proto/YWkKvsYgeHJskPsfuIpy7w/Codat-x-Generic-bank---Consent-Journey?page-id=601%3A4488&type=design&node-id=641-11421&viewport=1275%2C-2886%2C0.1&t=rrDznIIhmQ8EayyY-1&scaling=scale-down&starting-point-node-id=641%3A11421&show-proto-sidebar=1&mode=design)

Reviewer: ```suggestion
```
